src/databases/postgres.js

```diff
@@ -4,11 +4,19 @@ import dotenv from 'dotenv';
 dotenv.config();
 
 const { Pool } = pg;
+
+const user = 'postgres';
+const password = '123456';
+const host = 'localhost';
+const port = 5432;
+const database = 'linkrteste';
+
 const connection = new Pool({
-  connectionString: process.env.DATABASE_URL,
```

Comment:
You are removing the environment specific configurations here and exposing you local database settings, please return this to the original and configure your .env file to connect trough a connection string.